CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Everyone involved in [Carpentry@UiO](https://governance.readthedocs.io/en/latest/)
+agrees to abide by our [code of conduct](https://governance.readthedocs.io/en/latest/intro.html#code-of-conduct).
+
+## How to Contribute
+
+The easiest way to get started is to [create an issue](https://github.com/uio-carpentry/organisational/issues)
+to tell us about a spelling mistake,
+some awkward wording,
+or a factual error.
+This is a good way to introduce yourself
+and to meet some of our community members.
+
+1.  If you do not have a [GitHub](https://github.com) account,
+    you can [send us comments by email](mailto:contact-us@swcarpentry.uio.no).
+    However,
+    we will be able to respond more quickly if you use one of the other methods described below.
+
+2.  If you have a [GitHub](https://github.com) account,
+    or are willing to [create one](https://github.com/join),
+    but do not know how to use Git,
+    you can report problems or suggest improvements by [creating an issue](https://github.com/uio-carpentry/organisational/issues).
+    This allows us to assign the item to someone
+    and to respond to it in a threaded discussion.
+
+3.  If you are comfortable with Git,
+    and would like to add or change material,
+    you can submit a pull request (PR).
+    Instructions for doing this are [included below](#using-github).
+
+## Using GitHub
+
+If you choose to contribute via GitHub,
+you may want to look at
+[How to Contribute to an Open Source Project on GitHub][how-contribute].
+In brief:
+
+1.  We use [GitHub flow][github-flow] to manage changes:
+    1.  Fork the repository into your account and clone it to your desktop
+    1.  Create a new branch in your desktop clone of your fork for each significant change
+    2.  Commit the change in that branch
+    3.  Push that branch to your fork of the repository on GitHub
+    4.  Submit a pull request from that branch to [this repository](https://github.com/uio-carpentry/organisational)
+    5.  If you receive feedback,
+        make changes on your desktop and push to your branch on GitHub -
+        the pull request will update automatically.
+
+2.  The published copy of the material is in the `master` branch of [this repository](https://github.com/uio-carpentry/organisational)
+    1. Please keep the master branch of your fork up to date with the one from [this repository](https://github.com/uio-carpentry/organisational)
+    2. Create all branches from that up to date master branch in your fork
+    3. Please do *not* work directly in your fork's `master` branch,
+    since that will make it difficult for you to work on other contributions
+
+Staff from the UiO library as well as some of the Carpentry@UiO volunteers
```

Comment:
"The Carpentry@UiO coordinators as well as some community volunteers"

Staff from library sounds… weird :)